frontend/src/components/layout/Header/ErrorModal.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { Row, Col, Modal, Button } from 'react-bootstrap';
+import { IGenericNetworkAction, clear } from 'actions/genericActions';
+
+export interface IErrorModalProps {
+  // An array of network action errors.
+  errors: IGenericNetworkAction[];
+  // Whether to show the modal.
+  show?: boolean;
+  // Change the visible state.
+  setShow: (show: boolean) => void;
+}
+
+/**
+ * ErrorModal component that displays an error message and information in a modal.
+ * @param param0 ErrorModal component properties.
+ * @param param0.errors An array of errors.
+ * @param [param0.show] Whether to show the model.
+ * @param param0.setShow A function to set the show value.
+ * @returns ErrorModal component.
+ */
+export const ErrorModal = ({ errors, show, setShow }: IErrorModalProps) => {
```

Comment:
Moved this into its own file.  I would have preferred to have the `show, setShow` as internal properties.  But I couldn't figure out a good way to control when to show/hide it if I did that.

Reply:
While I think this is a good change to have this in its own file, does this have anything to do with this PR? This doesn't seem related to tenant configuration.

Reply:
Not related to tenant, but some tech-debt that is small enough and touches the components I'm working on.  I could have done two PRs, one for the config and then one for the tech-debt, but that would be more effort.  Not sure the value of the extra effort is critical in this case.  The only functional change is to dynamically display the appropriate title based on the tenant config.